.github/workflows/games.yml

```diff
@@ -37,4 +37,4 @@ jobs:
           platforms: linux/amd64
           push: true
           tags: |
-            ghcr.io/software-noob/pterodactyl-images:games_${{ matrix.tag }}
+            ghcr.io/Prefix/pterodactyl-images:games_${{ matrix.tag }}
```

Comment:
This shouldn't be changed and would be a breaking change either way.

Should you want to host it on your own repository, be aware that ghcr usernames must be lowercase.